### PR TITLE
[14.0][IMP] rma: add lot/serial in report

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Add Lot / Serial number in the RMA report.

@ForgeFlow

### CODE REVIEW

File: rma/report/rma_report_templates.xml [hunk 1]
```diff
@@ -84,19 +84,23 @@
                 <table class="table table-condensed">
                     <thead>
                         <tr>
-                            <th>Product</th>
-                            <th class="text-right">Quantity</th>
-                            <th class="text-right">Unit Price</th>
+                            <th t-name="product" invisible="1">Product</th>
```
Comment (LoisRForgeFlow) [inline rma/report/rma_report_templates.xml]: t-name is a directive in qweb reserved for putting a name to a template, this is not a correct use. To improve extensibility and ease xpath, you can simply use `name`

Comment (DavidJForgeFlow) [reply rma/report/rma_report_templates.xml]: Oh, I didn't know it, updated! Thx Lois!

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@DavidJForgeFlow Thanks! Can you fwport to v15 and 16?

**Comment by DavidJForgeFlow** [comment]
> @DavidJForgeFlow Thanks! Can you fwport to v15 and 16?

Done!

**Pull request merged**